lua/cmake-tools/utils.lua

```diff
@@ -68,16 +68,41 @@ function utils.execute(executable, opts)
   -- save all
   vim.cmd("wall")
   -- print("EXECUTABLE", executable)
-  local set_bufname = "file " .. opts.bufname
-  local prefix = string.format("%s %d new", opts.cmake_console_position, opts.cmake_console_size)
-
-  utils.close_cmake_console();
-  vim.cmd(prefix .. " | term " .. executable)
-  vim.opt_local.relativenumber = false
-  vim.opt_local.number = false
-  vim.cmd(set_bufname)
-  vim.bo.buflisted = false
-  vim.cmd("startinsert")
+  if const.cmake_launch_from_built_binary_directory == true then
```

Comment:
No need to check `== true`, just use `if const.cmake_launch_from_built_binary_directory`